EPAS - Added content as per DF-285 for EPAS 10 to 14 versions (#3059)
EnterpriseDB/docs: product_docs/docs/epas/14/epas_inst_windows/installing_advanced_server_with_the_interactive_installer/index.mdx

`--no-redwood-compat`

> Include the `--no-redwood-compat` keywords to instruct the server to create the cluster in PostgreSQL mode. When the cluster is created in PostgreSQL mode, the name of the database superuser is `postgres` and the name of the default database is `postgres`. The few Advanced Server’s features compatible with Oracle databases will be available with this mode. However, we recommend using the Advanced server in redwood compatibility mode to use all its features.

Reviewer: ```suggestion
Include the `--no-redwood-compat` keywords to instruct the server to create the cluster in PostgreSQL mode. When the cluster is created in PostgreSQL mode, the name of the database superuser is `postgres` and the name of the default database is `postgres`. A small subset of Advanced Server features compatible with Oracle databases are available with this mode. However, we recommend using Advanced Server in redwood compatibility mode to have access to all its features.
```
